__tests__/__action_fixtures__/subjectTemplate-suppressible.js

```diff
@@ -18,7 +18,7 @@ const subjectTemplate = build.subjectTemplate({
       uris: {
         "http://sinopia.io/testing/Suppressible/property1": "Property1",
       },
-      valueSubjectTemplateKeys: ["resourceTemplate:testing:uri"],
+      valueSubjectTemplateKeys: ["resourceTemplate:testing:suppressedUri"],
```

Comment:
I switched around two template fixtures, so a handful of changes like this where they were used.